File: BedAndBroadband/BedAndBroadband.Models/Hotel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BedAndBroadband.Data
{
    public class Hotel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public string Postcode { get; set; }
        public string Country { get; set; }
        public bool Wired { get; set; }
        public bool Wireless { get; set; }
    }
}

File: BedAndBroadband/BedAndBroadband/App/Models/hotel.ts
module Models {
    export interface Hotel {
        id: number;
        name: string;
        address: string;
        city: string;
        postcode: string;
        country: string;
        wired: bool;
        wireless: bool;
        lastRatingDate: Date;
        ratings: Rating[];
        averageQuality?: number;
        averageUpload?: number;
        averageDownload?: number;
        links?: Hypermedia.ILink[];
    }

    export interface Rating {
        id?: number;
        quality: number;
        downloadMbps: number;
        uploadMbps: number;
        date?: Date;
    }
}